### PR TITLE
Clarify INSTALL file according to the current state

### PR DESCRIPTION
**Pull request by otyshchenko1**
1. Describe possibility to build Graphic DDK from sources in the context
   of Gen3-test build procedure.
2. Clarify Gen3-test build steps to follow build in one-shot.

Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -90,24 +82,26 @@ For example (M3ULCB case),
 ln -s rcar-proprietary-graphic-salvator-x-m3-xt-domd.tar.gz rcar-proprietary-graphic-m3ulcb-xt-domd.tar.gz
 ln -s rcar-proprietary-graphic-salvator-x-m3-xt-domu.tar.gz rcar-proprietary-graphic-m3ulcb-xt-domu.tar.gz
 
-7. Edit local.conf:
-
-workspace_base/build/conf/local.conf
+5. Run the build script for current stable release:
```
Comment (andr2000) [inline INSTALL]: Not sure you need to mention "stable release" here

File: INSTALL [hunk 2]
```diff
@@ -90,24 +82,26 @@ For example (M3ULCB case),
...
 
-and append the following:
+python ./build_prod.py --build-type dailybuild --machine MACHINE_NAME --product PRODUCT_NAME --with-local-conf --config xt-gen3-test.cfg --with-do-build
 
-XT_GUESTS_INSTALL = "domu"
-XT_GUESTS_BUILD = "domu"
-XT_RCAR_EVAPROPRIETARY_DIR = "/path/to/eva/archives"
+Where the PRODUCT_NAME depends on whether prebuilt EVA proprietary packages are used or not:
+- gen3-test (to use prebuilt EVA proprietary packages)
+- gen3-test-src (to build Graphic DDK from sources)
```
Comment (andr2000) [inline INSTALL]: s/Graphic/Graphics/

File: INSTALL [hunk 3]
```diff
@@ -230,7 +224,7 @@ workspace_base/build/tmp/log/domd-image-weston/cooker/MACHINE-NAME-xt
 - DomU:
 workspace_base/build/tmp/log/domu-image-weston/cooker/MACHINE-NAME-xt
 
-10. If one wants to build any domain's images by hand, at the time of development
+7. If one wants to build any domain's images by hand, at the time of development
```
Comment (andr2000) [inline INSTALL]: s/by hand/manually/

Comment (andr2000) [reply INSTALL]: s/domain's/domain/

File: INSTALL (hunk 1 above)
Comment (otyshchenko1) [reply INSTALL]: ok

File: INSTALL (hunk 2 above)
Comment (otyshchenko1) [reply INSTALL]: ok

File: INSTALL (hunk 3 above)
Comment (otyshchenko1) [reply INSTALL]: ok

**Pull request merged**